### DISCUSSION TITLE
Issue getting docker working on new machine

### DISCUSSION THREAD

**Discussion by AdamXweb** [searx/searx#2754] [Installation issue: Docker]
Hi,
On a new AWS EC2 Ubuntu image I thought I'd test the docker instance.
I've followed the instructions to pull the repo and start the docker image.
The `.env` is configured with a hostname and e-mail only.

On `sudo ./start.sh`
The URL loaded shows 'Internal Server Error', and looking at the docker logs, Morty keeps restarting with error:
`2021/01/22 05:59:01 Error parsing -keyillegal base64 data at input byte 4`

I haven't modified anything else apart from above and the docker-compose to log the error with`driver: json-file`

Can provide additional info

**Comment by AdamXweb** [comment]
update: by running `openssl rand -base64 33` and placing result as the key solved the morty issue, however page still loads with 'Internal Server Error'

**Comment by selleronom** [comment]
remove the directory "/usr/local/searx-docker/searx" (created after starting the containers) and restart the services.
